conans/test/functional/build_helpers/autotools_environment_test.py

```diff
@@ -55,15 +53,18 @@ def package_id(self):
                      "hello.h": header,
                      "hello.cpp": body})
         client.run("create . danimtb/testing")
-        pref = PackageReference.loads("test/1.0@danimtb/testing:%s" % NO_SETTINGS_PACKAGE_ID)
-        pkg_path = client.cache.package_layout(pref.ref).package(pref)
+        pref = client.get_latest_prev(ConanFileReference.loads("test/1.0@danimtb/testing"),
```

Comment:
Tip: allow ``TestClient.get_latest_prev()`` to accept a string too, and convert internally to ``ConanFileFerence``. (keep the possibility of passing a ConanFileReference)
TestClient helpers are there to make test code simpler.